mypyc/ops.py

```diff
@@ -1303,6 +1303,8 @@ def __init__(self, name: str, module_name: str, is_trait: bool = False) -> None:
         self.name = name
         self.module_name = module_name
         self.is_trait = is_trait
+        self.properties = OrderedDict()  # type: OrderedDict[str, RType]
+        self.property_getters = OrderedDict()  # type: OrderedDict[str, FuncIR]
```

Comment:
Document the relationship between properties and methods in the ClassIR data structures.